docs/registry/create/_index.md

```diff
@@ -834,6 +1196,17 @@ fn (c *component) someFunction(a int) {
 }
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+Messages sent to `std::cout` in your C++ code are automatically sent to the Viam app over gRPC when a network connection is available.
```

Comment:
This is not exactly true, and apologies for my probably misleading explanation. `std::cout` messages certainly go to STDOUT. The RDK gathers all the STDOUT output from the process and logs it (no gRPC involved). We will soon have a custom C++ logger that logs over gRPC back to the parent when possible and logs to STDOUT otherwise. But, `std::cout` messages will never be sent back over gRPC, only messages logged through that not-yet-existing logger.

Reply:
Thanks! Removed mention of `grcp` 👍 Happy to circle back to this  later and update when you make the upcoming custom logger change!